hivemind_etl/mediawiki/wikiteam_crawler.py

```
@@ -0,0 +1,83 @@
+
+from wikiteam3.dumpgenerator import DumpGenerator
+
+
+class WikiteamCrawler:
+    def __init__(
+        self,
+        community_id: str,
+        xml: bool = True,
+        force: bool = True,
+        curonly: bool = True,
+        namespaces: list[int] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)